02-molecules/cards/Card/Card.component.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Image from '../../../01-atoms/images/ResponsiveImage/ResponsiveImage.component';
+import { imageTypeProps } from '../../../01-atoms/images/Image/Image.component';
+import bem from '../../../_utils/bem';
+import Heading from '../../../01-atoms/text/Heading.component';
+import Paragraph from '../../../01-atoms/text/Paragraph.component';
+import Button from '../../../01-atoms/buttons/Button.component';
+
+const Card = ({
+  outputImage,
+  image,
+  heading,
+  subheading,
+  body,
+  buttonText,
+  cardModifiers,
+}) => {
+  const block = 'card';
+  return (
+    <div className={bem(block, '', cardModifiers)}>
+      {/* Image */}
+      <div className={bem(block, 'image')}>
+        {image && <Image outputImage={outputImage} image={image} />}
+      </div>
+      {/* Content */}
+      <div className={bem(block, 'content')}>
+        {/* Heading */}
+        {heading && (
+          <Heading
+            level="2"
+            children={heading}
+            block={block}
+            element="heading"
+          />
+        )}
+        {/* Subheading */}
+        {subheading && (
+          <Heading
+            level="3"
+            children={subheading}
+            block={block}
+            element="subheading"
+          />
+        )}
+        {/* Body */}
+        {body && <Paragraph children={body} block={block} element="body" />}
+        {/* Button */}
+        {buttonText && (
+          <Button children={buttonText} block={block} element="button" />
+        )}
+      </div>
+    </div>
+  );
+};
+
+Card.propTypes = {
+  outputImage: PropTypes.bool,
+  image: imageTypeProps,
+  heading: PropTypes.string,
+  subheading: PropTypes.string,
+  body: PropTypes.string,
```

Comment:
Not sure this is what you will want. Using the property `body` as a string means that anything passed through here will render as a string. Formatting, like strong or emphasized text can not be passed through string and rendered in the virtual DOM. Instead, use {children} if you want more than strings.

Reply:
So, like `body: PropTypes.children`?

Reply:
Or something like this: https://reactjs.org/docs/typechecking-with-proptypes.html#requiring-single-child

e.g.
```
    // This must be exactly one element or it will warn.
    const body = this.props.children;

...
        {body && (
          <Paragraph block={block} element="body">
            {body}
          </Paragraph>
        )}
...

	body: PropTypes.element
```

Reply:
I believe `children` is the better solution as it's more flexible then `element`.

```
const Card = ({
  outputImage,
  image,
  heading,
  subheading,
  children,  //Change body to children
  linkText,
  linkUrl,
  cardModifiers,
}) => {
...
        {body && (
          <Paragraph block={block} element="body">
            {children}
          </Paragraph>
        )}
...
body: PropTypes.children
```

Or if you want to continue to call it body, you can give it an alias when restructuring it. I believe the syntax is: 
```
children as body
```

Reply:
That makes sense